ci-operator/config/openshift/release/openshift-release-master__nightly-4.16.yaml

```diff
@@ -827,6 +827,20 @@ tests:
       - observers-resource-watch
     workflow: telco5g-hcp-cnf
 - as: metal-ovn-network-flow-matrix-bm
+  cluster: build05
+  cron: 0 23 * * 1
+  steps:
+    cluster_profile: equinix-ocp-metal
+    env:
+      CLUSTERTYPE: baremetal
```

Comment:
This causes the job to use a full baremetal cluster in our lab, it isn't using equinix
can you remove it so that the job is using a virtualized setup?